src/libcrun/status.c

```diff
@@ -214,6 +214,7 @@ libcrun_container_delete_status (const char *state_root, const char *id, libcrun
   if (UNLIKELY (ret < 0))
     return crun_make_error (err, errno, "cannot rm state directory '%s/%s'", dir, id);
 
+  closedir(d);
```

Comment:
instead of closing the dir here.

Let's initialize `d` at line 183 as: `cleanup_dir DIR *d = NULL;`

Reply:
@jnovy could you amend this change and push a new version?

Reply:
Sure thing @giuseppe - let me drop this PR and I will do another one.

Reply:
are you going to open a new PR? :-)